libvast/src/format/zeek.cpp

```diff
@@ -445,18 +445,22 @@ caf::error reader::parse_header() {
   VAST_DEBUG(this, "    #unset_field", unset_field_);
   VAST_DEBUG(this, "    #path", path);
   VAST_DEBUG(this, "    #fields:");
-  for (auto i = 0u; i < layout_.fields.size(); ++i)
-    VAST_DEBUG(this, "     ", i << ')',
-               layout_.fields[i].name << ':', layout_.fields[i].type);
   // If a congruent type exists in the schema, we give the schema type
   // precedence.
-  if (auto t = schema_.find(path))
-    if (t->name() == path) {
-      if (congruent(type_, *t))
-        type_ = *t;
-      else
-        return make_error(ec::format_error, "incongruent types in schema");
+  if (auto t = schema_.find(layout_.name())) {
+    auto r = caf::get_if<record_type>(t);
+    if (!r)
+      return make_error(ec::format_error, "record_type required");
```

Comment:
Since error messages are user facing, I think this should be reworded not to mention the internal concept of a `record_type`.

Reply:
`record` is a context dependent keyword in the schema language. But I will reword this message to be more informative and understandable.